docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -2,81 +2,110 @@
+#### Set an initial pose for the ego vehicle
 
-   ![set-goal-pose](images/planning/lane-following/set-goal-pose.png)
+![set-initial-pose](images/planning/lane-following/set-initial-pose.png)
 
-4. To place a dummy object such as a pedestrian, click the `2D Dummy Pedestrian` button in the toolbar.
+1. Click the `2D Pose estimate` button in the toolbar, or hit the `P` key.
+2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the initial pose. An image representing the vehicle should now be displayed.
 
-   ![set-dummy-pedestrian](images/planning/lane-following/set-dummy-pedestrian.png)
+!!! warning
 
-5. Engage the ego vehicle.
+    Remember to set the initial pose of the car in the same direction as the lane. To confirm the direction of the lane, check the arrowheads displayed on the map.
 
-   ```bash
-   source ~/autoware/install/setup.bash
-   ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: true" -1
-   ```
+#### Set a goal pose for the ego vehicle
 
-   ![start-driving](images/planning/lane-following/start-driving.png)
+1. Click the `2D Goal Pose` button in the toolbar, or hit the `G` key.
+2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose. If done correctly, you will see a planned path from initial pose to goal pose.
 
-6. To delete objects, click the `Delete All Objects` button in the toolbar.
+![set-goal-pose](images/planning/lane-following/set-goal-pose.png)
 
-   ![delete-objects](images/planning/lane-following/delete-objects.png)
+#### Engage the ego vehicle
 
-7. After that, the vehicle will restart driving and reach the goal.
+Now you can start the ego vehicle driving by clicking the `Engage` button in `AutowareStatePanel`. Alteratively, you can manually engage the vehicle by running the following command:
 
-   ![restart-driving](images/planning/lane-following/restart-driving.png)
+```bash
+source ~/autoware/install/setup.bash
+ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: true" -1
+```
 
-   ![reach-goal](images/planning/lane-following/reach-goal.png)
+![start-driving](images/planning/lane-following/engage-and-start-planning.png)
 
-## How to simulate parking maneuvers
+### Parking maneuvers
 
 1. Set an initial pose and a goal pose, and engage the ego vehicle.
 
    ![after-set-goal-pose](images/planning/parking/after-set-goal-pose.png)
 
 2. When the vehicle approaches the goal, it will change to the parking mode.
+3. After that, the vehicle will start driving backwards and park at the destination parking spot.
 
-   ![change-to-parking-mode](images/planning/parking/change-to-parking-mode.png)
+   ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
-3. After that, the vehicle will start driving backwards and park at the destination parking spot.
+## Advanced Simulations
+
+### Dummy objects
```

Comment:
Does it make it more clear to call this tutorial "Placing dummy objects"? Or is "Adding dummy objects" better?

```suggestion
### Placing dummy objects
```

Reply:
"Placing" would be good for the user